src/main/java/bio/terra/common/iam/AuthenticatedUserRequest.java

```diff
@@ -14,6 +14,12 @@
   private final String subjectId;
   private final String token;
 
+  protected AuthenticatedUserRequest(Builder builder) {
```

Comment:
nit: if we're going to move the constructor, let's make it `private` and even make the class final. `protected` tells me we want subclasses, but we don't want subclasses by default

Reply:
Done